ert_storage/endpoints/priors.py

```diff
@@ -0,0 +1,53 @@
+from uuid import UUID
+from typing import Mapping, Type
+from fastapi import APIRouter, Depends
+from ert_storage.database import Session, get_db
+from ert_storage import database_schema as ds, json_schema as js
+from ert_storage.json_schema.prior import (
+    ConstPrior,
+    TriangularPrior,
+    NormalPrior,
+    LognormalPrior,
```

Comment:
My eyes really want this to be LogNormalPrior

Reply:
What do you think about the other names? It seems like I'm mixing and matching short and long names. Ie, `ConstPrior` and not `ConstantPrior`; `TriangularPrior` and not `TrigPrior`.

This could be `LogNormPrior` or `LogarithmicNormalPrior`.

Reply:
hmm... I prefer having the same yes. If we should go for long or short --- i'm not sure 🤔  I can say it is not often  long names makes me wonder what they mean - but sometimes they are annoying because they are almost snake_cased sentences. On the other hand - short names makes me often stop and think what they mean, but they fits nicely into the code.... I would say it is a trade-off, and - definitely i would say `ConstantPrior` over `ConstPrior`. `Log` is quite normal being used as only that word so i guess both `LogNormalPrior` or `LogarithmicNormalPrior` would be fine, with the first one easier to both read and write.